spec/openapi.yaml

```
@@ -163,18 +163,18 @@ paths:
           $ref: "#/components/responses/404-object-by-id-not-found"
       tags:
         - TEA Release
-  /release/{uuid}/collection/{version}:
+  /release/{uuid}/collection/{collectionVersion}:
     get:
-      description: Get all versions of a TEA Collection by its UUID
-      operationId: getTeaCollectionVersionByCollectionIdAndVersion
+      description: Get a specific Collection (by version) for a TEA Release by its UUID
+      operationId: getTeaCollectionVersionByReleaseId
```

```suggestion
      operationId: getCollection
```

_Nit_: Since UUID and `version` are the primary key for a TEA Collection, I don't think we need the `by*` part.

**Note**: The name of this operation should lose `Tea` or the previous one should gain one.